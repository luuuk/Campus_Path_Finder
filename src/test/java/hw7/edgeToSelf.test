# This test checks the output of the path-finding algorithm when the start and end nodes are the same

LoadGraph test test.tsv

FindPath test a a