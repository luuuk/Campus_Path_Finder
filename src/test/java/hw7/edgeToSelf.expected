# This test checks the output of the path-finding algorithm when the start and end nodes are the same

loaded graph test

path from a to a:
total cost: 0.000
